ros/src/twist_controller/twist_controller_mod.py

```diff
@@ -102,13 +104,21 @@ def control(self, target_linear_velocity, target_angular_velocity, current_linea
             control_mode = "Manual braking"
         elif velocity_error < -1 * limit_constant * current_linear_velocity:
```

Comment:
Changing this line to:
`elif velocity_error < -1 * max(limit_constant * current_linear_velocity, 0.1):`
prevents the error threshold from getting too small near zero and seems to trigger the manual brake filter reset at each standing start.

Reply:
Great. Let me patch this in tomorrow and give it a try. Thanks!

Reply:
Tested and works well.